tests/test_interface.rb

```diff
@@ -1043,7 +1043,7 @@ def test_interface_ipv4_arp_timeout
 
   def test_interface_ipv4_forwarding
     intf = interfaces[0]
-    interface_ethernet_default(intf)
+    interface_ethernet_default(interfaces_id[0])
     i = Interface.new(intf)
```

Comment:
I don't understand this change. What's the difference between what you get from `interfaces[0]` and `interfaces_id[0]`?
If `interfaces[0]` is a problem for `interface_ethernet_default` then I would expect the `Interface.new()` call to have a problem as well...?